pybaseball/batting_and_pitching_splits

```diff
@@ -0,0 +1,139 @@
+import requests
+import pandas as pd
+import bs4 as bs
+import concurrent.futures   
+
+def download_url(url):
+    """
+    Creates a local file with the contents of each table
+    """
+    resp=requests.get(url)
+    title="".join(x for x in url if x.isalpha())+"html"
+    
+    with open(title, "wb") as fh:
+        fh.write(resp.content)
+    
+def download_tables(table_urls):
+    threads=min(30, len(table_urls))
+    
+    with concurrent.futures.ThreadPoolExecutor(max_workers=max(1,threads)) as executor:
+        executor.map(download_url, table_urls)
+        
+
+def read_split_files(urls):
+    """
+    reads the split files that were created and outputs a dataframe of stats by split. 
+    """
+    html=[]
+    data=[]
+    for i in range(len(urls)):
+        with open("".join(x for x in urls[i] if x.isalpha())+"html") as f:
+            html=f.read()
+        soup=bs.BeautifulSoup(html, 'lxml')
+        playerid=urls[i].split('%3D')[1].split('%26year')[0]
+        split=urls[i].split('div_')[-1]
+        table = soup.find_all('table')[0]
+        headings = [th.get_text() for th in table.find("tr").find_all("th")][1:]
+        headings.append('Split Type')
+        headings.append('Player ID')
+        headings.append('1B') #singles data isn't included in the tables so this appends the column header
+        data.append(headings)
+        table=soup.find('div')
+        table_body = table.find('tbody')
+        rows = table_body.find_all('tr')
+        for row in rows:
+            cols = row.find_all('td')
+            cols = [ele.text.strip() for ele in cols]
+            cols.append(split)
+            cols.append(playerid)
+            data.append([ele for ele in cols])
+    data = pd.DataFrame(data)
+    data = data.rename(columns=data.iloc[0])
+    data = data.reindex(data.index.drop(0))
+    data = data.set_index(['Player ID', 'Split Type', 'Split'])
+    data = data.drop(index=['Split'], 
+                            level=2)
+    data = data.apply(pd.to_numeric, errors='coerce')
+    data = data.fillna(0.0)
+    data = data.iloc[:,:-3]
```

Comment:
this line needs a comment -- what's this doing?

Reply:
removed that line. It was no longer needed.